gapic/src/main/com/google/gapid/views/PerformanceView.java

```diff
@@ -119,6 +126,13 @@ public PerformanceView(Composite parent, Models models, Widgets widgets) {
     filterButton.setImage(widgets.theme.more());
     filterButton.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false));
 
+    if(ProfileExperiments.SHOULD_ENABLE_EXPERIMENTS) {
```

Comment:
This button should be before the filter button, since the filter button goes along with the presets.